packages/base/src/toolbar/widget.tsx

```diff
@@ -33,6 +33,15 @@ export class ToolbarWidget extends Toolbar {
     this.addClass('jGIS-toolbar-widget');
 
     if (options.commands) {
+      this.addItem(
+        'zoomColor',
```

Comment:
I am a bit confused by the name "zoomColor" and how you are using your color stops.

- I understood that the color stop widget was for defining color stops (and interpolate between those) given a data channel from the geotiff. I don't see in your implementation how the data channel is being used? 
- I also don't understand how it's related to zoom, is it a name from an old implementation?

Reply:
As discussed in-person:

>  I also don't understand how it's related to zoom, is it a name from an old implementation?

It is indeed a name from an old implementation

> I don't see in your implementation how the data channel is being used?

This is not yet fully exposed to the UI, we will need to iterate on the UI to be able to select bands and apply styling to them